### PR TITLE
fix: move card styles from ngx to styles

### LINKED ISSUE #1729: negative numbers in RTL are incorrect
**Describe the bug**
Right now in RTL mode negative numbers are incorrect (minus goes after number)
It's needed to have a predefined wrapper or class to keep numbers (potentially phone numbers as well) in LTR even when parent is in RTL.

This problem can be seen in `object-number` and `card` examples.
Analytical Card:
image
Object Number:
image

> Is this issue related to a specific component?
No
> What version of the Fundamental Library Styles are you using?
0.12

**Expected behavior**
Negative numbers should remain in LTR

### PR DESCRIPTION
**Pull request by DeepakSap14**
## Related Issue
closes #1729 

## Description
Currently fundamental-ngx card component has some styles present in fundamental-ngx, which can be moved to fundamental-styles.

### Before:
Screenshot 2021-04-07 at 2 29 39 PM

Screenshot 2021-04-07 at 2 29 13 PM

### After:
Negative sign placed correctly
Screenshot 2021-04-07 at 2 32 39 PM

Table border removed
Screenshot 2021-04-07 at 2 35 36 PM

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- na Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/card.scss [hunk 1]
```diff
@@ -231,6 +231,11 @@ $fd-card-default-body-background-color: var(--sapTile_Background) !default;
     @include fd-reset();
 
     color: var(--sapTile_TitleTextColor);
+
+    @include fd-rtl() {
+      direction: ltr;
```
Comment (InnaAtanasova) [inline src/card.scss:236]: why are you adding this?

File: src/card.scss [hunk 2]
```diff
@@ -364,4 +371,23 @@ $fd-card-default-body-background-color: var(--sapTile_Background) !default;
...
+    @include fd-reset();
+
+    flex: auto;
+  }
+
+  .#{$numeric-content}__kpi {
+    @include fd-reset();
+
+    font-size: 2.25rem;
+  }
```
Comment (InnaAtanasova) [inline src/card.scss:392]: You can't have `numeric-content` classes inside Card component, this is violating the self-contained rules.

File: src/card.scss [hunk 3]
```diff
@@ -364,4 +371,23 @@ $fd-card-default-body-background-color: var(--sapTile_Background) !default;
...
+    @include fd-reset();
+    @include fd-flex-center();
+
+    min-height: 5rem;
+  }
+
+  .#{$numeric-content}__scale-container {
+    @include fd-reset();
+
+    flex: auto;
```
Comment (InnaAtanasova) [inline src/card.scss:385]: `flex: auto` is not working properly in IE11. Have you checked it?

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

**Comment by DeepakSap14** [comment]
@InnaAtanasova please check this PR along with PR https://github.com/SAP/fundamental-ngx/pull/5015.
on closer looking, some css changes present in #5015 was not needed and already proper class is available. so i have used that.
rest of css changes are migrated here.

File: src/card.scss (hunk 2 above)
Comment (DeepakSap14) [reply src/card.scss]: this class is not needed. now using correct css class in ngx.

File: src/card.scss (hunk 3 above)
Comment (DeepakSap14) [reply src/card.scss]: removed.

File: src/numeric-content.scss [hunk 4]
```diff
@@ -56,6 +56,11 @@ $fd-numeric-content-elements-spacing-s: 0.5rem !default;
 
   .#{$block}__kpi {
     font-size: 1.5rem;
+
+    @include fd-rtl() {
+      direction: ltr;
+      text-align: right;
+    }
```
Comment (InnaAtanasova) [inline src/numeric-content.scss:63]: would be good if this is moved to another mixin for future re-use

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
Please trigger Chromatic checks

**Review by droshev — APPROVED** [review]

**Pull request merged**